snuba/cli/bulk_load.py

```diff
@@ -0,0 +1,36 @@
+import logging
+
+import click
+
+from snuba import settings
+from snuba.datasets.factory import get_dataset
+from snuba.snapshots.postgres_snapshot import PostgresSnapshot
+
+
+@click.command()
+@click.option('--dataset', type=click.Choice(['events', 'groupedmessage', 'outcomes']),
+              help='The dataset to bulk load')
+@click.option('--source',
+              help='Source of the dump. Depending on the dataset it may have different meaning.')
+@click.option('--dest-table',
+              help='Clickhouse destination table.')
+@click.option('--log-level', default=settings.LOG_LEVEL, help='Logging level to use.')
+def bulk_load(dataset, dest_table, source, log_level):
+    import sentry_sdk
+
+    sentry_sdk.init(dsn=settings.SENTRY_DSN)
+    logging.basicConfig(level=getattr(logging, log_level.upper()), format='%(asctime)s %(message)s')
+
+    logger = logging.getLogger('load-snapshot')
```

Comment:
Pretty much every other logger is prefixed with `snuba.*`:

```
snuba/api.py:logger = logging.getLogger('snuba.api')
snuba/cleanup.py:logger = logging.getLogger('snuba.cleanup')
snuba/clickhouse.py:logger = logging.getLogger('snuba.clickhouse')
snuba/consumer.py:logger = logging.getLogger('snuba.consumer')
snuba/datasets/events_processor.py:logger = logging.getLogger('snuba.processor')
snuba/migrate.py:logger = logging.getLogger('snuba.migrate')
snuba/optimize.py:logger = logging.getLogger('snuba.optimize')
snuba/perf.py:logger = logging.getLogger('snuba.perf')
snuba/replacer.py:logger = logging.getLogger('snuba.replacer')
snuba/state.py:logger = logging.getLogger('snuba.state')
snuba/util.py:logger = logging.getLogger('snuba.util')
snuba/writer.py:logger = logging.getLogger('snuba.writer')
```

Reply:
ok